modules/bluetoothid.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#print id code of bluetooth, subfunction prints hostname and bluetooth id
+
+function bluetoothid () {
+  #check if bluetooth has an id
+  btidfile=/etc/bluetooth-id
+  if [ ! -f "${btidfile}" ]; then
+    echo "No ID. Bluetooth service is not on."
+    return 1
+  fi
+
+  bid=$(cat ${btidfile})  #get id of the bluetooth
+  nname=$(uname -n)  #get network name
+
+  case "$1" in
+    "")
+      echo "Bluetooth ID is: ${bid}"
+      ;;
+    "hostid")
+      echo "Bluetooth's Network Host ID is: ${nname}-${bid}"
+      ;;
+  esac
```

Comment:
*)
 echo "Error: please leave empty argument or type in hostid"
;;

Reply:
I was thinking the same. Wasn't sure if it would be necessary.
@dogi, should this be added?